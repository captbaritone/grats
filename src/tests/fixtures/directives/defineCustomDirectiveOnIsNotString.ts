/**
 * This is my custom directive.
 * @gqlDirective on FIELD_DEFINITION This is a description with {@link https://example.com some link}
 * and `inline code` to demonstrate structured comments.
 */
function customDirective() {}
