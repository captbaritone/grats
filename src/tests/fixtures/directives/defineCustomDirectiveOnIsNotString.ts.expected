-----------------
INPUT
----------------- 
/**
 * This is my custom directive.
 * @gqlDirective on FIELD_DEFINITION This is a description with {@link https://example.com some link}
 * and `inline code` to demonstrate structured comments.
 */
function customDirective() {}

-----------------
OUTPUT
-----------------
src/tests/fixtures/directives/defineCustomDirectiveOnIsNotString.ts:3:65 - error: Expected Grats JSDoc tag value to be simple text.

3  * @gqlDirective on FIELD_DEFINITION This is a description with {@link https://example.com some link}
                                                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
